pkg/gadgets/oomkill/types/types.go

```diff
@@ -21,10 +21,10 @@ import (
 type Event struct {
 	eventtypes.Event
 
-	TriggeredPid  uint32 `json:"tpid,omitempty"`
-	TriggeredComm string `json:"tcomm,omitempty"`
-	KilledPid     uint32 `json:"kpid,omitempty"`
-	KilledComm    string `json:"kcomm,omitempty"`
+	TriggeredPid  uint32 `json:"fpid,omitempty"`
+	TriggeredComm string `json:"fcomm,omitempty"`
+	KilledPid     uint32 `json:"tpid,omitempty"`
```

Comment:
`tpid` was the tag of `TriggeredPid` and now it corresponds to `TriggeredPid`. Is that intentional?